src/components/Sidebar/SidebarContext.spec.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  SidebarContextProvider,
+  SidebarContextConsumer
+} from './SidebarContext';
+
+describe('SidebarContext', () => {
+  describe('styles', () => {
```

Comment:
I don't think a snapshot test makes sense here. Those are only really useful to test styles, which this component doesn't have.

Reply:
Makes sense to me! Thanks, Connor!